forge-gui/res/cardsfolder/upcoming/displacer_kitten.txt

```diff
@@ -0,0 +1,10 @@
+﻿Name:Displacer Kitten
+ManaCost:3 U
+Types:Creature Cat Beast
+PT:2/2
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ DBBlink | TriggerZones$ Battlefield | TriggerDescription$ Avoidance = Whenever you cast a noncreature spell, exile up to one target nonland permanent you control, then return it to the battlefield under its owner's control.
```

Comment:
Avoidance =
I think this is causing the test to freeze, should be:
Avoidance -